2014-15/B13 Create CHI - Deceased Lookup.sps

```diff
@@ -40,11 +35,11 @@ Numeric death_date (Date12).
 Compute death_date = death_date_CHI.
 If Not(sysmis(death_date_NRS)) death_date = death_date_NRS.
 
-save outfile = !Extracts_Alt + "All Deaths.zsav"
+save outfile = !Deaths_dir + "All Deaths.zsav"
     /zcompressed.
 
-get file = !Extracts_Alt +  "All Deaths.zsav".
+get file = !Deaths_dir +  "All Deaths.zsav".
```

Comment:
These files (B13) need moving to all_years since they're not year specific. Can use the same logic as with PC and GPprac, or just overwrite every update.

The file name could be changed to remove spaces e.g. all_deaths.zsav possibly using a !Deaths_file macro, which would also allow to easily use the !update macro (as with PC file).  If changing macros and/or names then Social Care and C06 also need updating as they reference the file.

Reply:
Deaths Lookup now moved to all years using the same logic as PC and GPprac lookups. No spaces in file name now and will save using !Deaths_dir located in SLF_Extracts/Deaths on hscdiip. Also made these changes on Social Care branch

Reply:
Great, looks good to me :)